### ISSUE TITLE
Keyboard autorepeat no longer working

### ISSUE THREAD

**Issue by MirrorPusher** [MirrorPusher/Stevie#47]
vorticon reported:
_I noticed key repeat was not implemented. Not a huge deal but again more of a convenience. This can come handy when moving the cursor around the page._

Keyboard autorepeat is no longer working after switching to KSCAN.
Bug reproduced in Stevie 1.2Q

**Comment by MirrorPusher** [comment]
I looked into the keyboard auto-repeat issue and did multiple tests and even did some changes. 
In the end I decided that I'm going to remove the autorepeat functionality in Stevie.
 
Reason is, that the OS monitor KSCAN only detects keyboard changes. You can read all about it on Thierry's site:
http://www.unige.ch/medecine/nouspikel/ti99/keyboard.htm#auto-repeat

The way they implemented this is actually a good way from my point of view, because you can get away with a lot of key debounce stuff.

The issue with those timed loops is that in the end, they slow down your program and you get strange results when you have a faster processor (thinking about classic99 CPU overdrive for example or some future HW developments).

There are ways to influence KSCAN as described in Thierry's site. In Stevie I tried that by running a task in the task scheduler that periodically clears the keyboard buffer (controlled by VDP interrupt flag). 

However as said, I came to the conclusion that you are opening a can of worms and the purpose of Stevie is to be fast and compatible with e.g. devices like USB keyboard adapters.

Long story short, no key autorepeat but you do get the benefit of a faster response and it works real nice in classic99 and on the TI-99/4a.
